plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverApplication.java

```diff
@@ -269,7 +269,9 @@ public Object start(IApplicationContext context) {
         // It sets main windows name and images
         getDisplay();
 
-        updateSplashHandler();
+        if (!RuntimeUtils.isMacOS() || !RuntimeUtils.isOSVersionAtLeast(14, 0, 0)) {
```

Comment:
Should be this. Can be read as "if not macOS, or macOS' version is at least 14, then update".
```suggestion
        if (!RuntimeUtils.isMacOS() || RuntimeUtils.isOSVersionAtLeast(14, 0, 0)) {
```